Assets/Plugins/iOS/Shopify/MessageCenter.swift

```diff
@@ -32,14 +32,14 @@ import PassKit
     private static var messages = [String: UnityMessage]()
     
     static func send(_ message: UnityMessage) {
-        UnityInterfaceWrapper.sendMessage(try! message.serializedString(), toObject: message.recipientObjectName, havingMethodName: message.recipientMethodName)
+        MessageSender.sendMessage(try! message.serializedString(), object: message.object, method: message.method)
     }
     
     static func send(_ message: UnityMessage, completionHandler: UnityMessage.MessageCompletion?) {
         
         messages[message.identifier] = message
         
-        UnityInterfaceWrapper.sendMessage(try! message.serializedString(), toObject: message.recipientObjectName, havingMethodName: message.recipientMethodName)
+        MessageSender.sendMessage(try! message.serializedString(), object: message.object, method: message.method)
```

Comment:
I was mislead when I got a compile error when trying to include the bridging header, so I thought It had something to do with recursive imports. But I've found the fix.